rotkehlchen/data/all_assets.json

```diff
@@ -7377,6 +7377,16 @@
         "symbol": "MCI",
         "type": "ethereum token"
     },
+    "MUST": {
+        "coingecko": "Must",
+        "cryptocompare": "",
+        "ethereum_address": "0x9c78ee466d6cb57a4d01fd887d2b5dfb2d46288f",
```

Comment:
The addresses must always be checksummed. Tests will thankfully fail and won't let you commit that.